custom/opm/reports.py

```diff
@@ -824,6 +824,36 @@ class MetReport(CaseReportMixin, BaseReport):
     cache_key = 'opm-report'
 
     @property
+    def row_objects(self):
+        """
+        Returns a list of objects, each representing a row in the report
+        """
+        rows = []
+        self._debug_data = []
+        for index, row in enumerate(self.get_rows(self.datespan), 1):
+            try:
+                rows.append(self.get_row_data(row, index=index))
+            except InvalidRow as e:
+                if self.debug:
+                    import sys
+                    import traceback
+                    type, exc, tb = sys.exc_info()
+                    self._debug_data.append({
+                        'case_id': row._id,
+                        'message': e,
+                        'traceback': ''.join(traceback.format_tb(tb)),
+                    })
+        return rows
```

Comment:
Actually, maybe that's not so bad - it would at least indicate that there are more cases there which aren't appearing.  You can disregard this comment.